components/gitpod-db/src/typeorm/workspace-cluster-db-impl.ts

```diff
@@ -42,6 +42,11 @@ export class WorkspaceClusterDBImpl implements WorkspaceClusterDB {
         return repo.findOne(name);
     }
 
+    async findAll(): Promise<WorkspaceCluster[]> {
+        const repo = await this.getRepo();
```

Comment:
I added this in addition to findFiltered because findFiltered does not return the tls config which we need to authenticate against the ws-manager endpoint

Reply:
Jep. We did so to reduce traffic (bc it turned out to be a perf problem). See [comment below](https://github.com/gitpod-io/gitpod/pull/11010#discussion_r911154433).